sql/6_0_0-to-6_1_0_upgrade.sql

```diff
@@ -687,3 +687,63 @@ ALTER TABLE `form_clinical_notes` ADD `note_related_to` TEXT COMMENT 'Reference
 #IfMissingColumn form_care_plan note_related_to
 ALTER TABLE `form_care_plan` ADD `note_related_to` TEXT COMMENT 'Reference to lists id for note relationships(json)';
 #EndIf
+
+#IfMissingColumn form_vitals uuid
+ALTER TABLE `form_vitals` ADD `uuid` binary(16) DEFAULT NULL AFTER `id`;
+#EndIf
+
+#IfMissingColumn uuid_mapping resource_path
+ALTER TABLE `uuid_mapping` ADD `resource_path` VARCHAR(255) DEFAULT NULL;
+#EndIf
+
+#IfMissingColumn form_vitals ped_weight_height
+ALTER TABLE `form_vitals` ADD `ped_weight_height` FLOAT(4,1) NULL DEFAULT '0.00';
+#EndIf
+
+#IfMissingColumn form_vitals ped_bmi
+ALTER TABLE `form_vitals` ADD `ped_bmi` FLOAT(4,1) NULL DEFAULT '0.00';
+#EndIf
+
+#IfMissingColumn form_vitals ped_head_circ
+ALTER TABLE `form_vitals` ADD `ped_head_circ` FLOAT(4,1) NULL DEFAULT '0.00';
+#EndIf
+
+#IfMissingColumn history_data uuid
+ALTER TABLE `history_data` ADD `uuid` binary(16) DEFAULT NULL AFTER `id`;
+#EndIf
+
+#IfNotIndex history_data uuid
+-- History table uses a historical record data pattern where the same record is duplicated multiple times to retain
+-- history, this makes it so the record has the same uuid.  We only index the record instead of keeping it UNIQUE
+CREATE INDEX `uuid` ON `history_data` (`uuid`);
+#EndIf
+
+#IfUuidNeedUpdate history_data
```

Comment:
@adunsulag , Just trying to figure out best way to go here since per above these are not going to be a uuid per row. If this is instead a unique uuid per patient, then would use the vertical table update mechanism (#IfUuidNeedUpdateVertical) with the pid as the grouping index.

Reply:
The problem is there is nothing unique in the history_data table so the vertical table update doesn't work as there is no true composite key where the combination of columns is unique.  What we may need to do is rearchitect the table structure and have a sub table such as history_data_archive or something like that if we truly need the historical audit of the data.